appinventor/components/src/com/google/appinventor/components/runtime/CoordinateData.java

```diff
@@ -0,0 +1,38 @@
+package com.google.appinventor.components.runtime;
+
+import com.google.appinventor.components.annotations.DesignerComponent;
+import com.google.appinventor.components.annotations.SimpleFunction;
+import com.google.appinventor.components.annotations.SimpleObject;
+import com.google.appinventor.components.common.ComponentCategory;
+import com.google.appinventor.components.runtime.util.YailList;
+import java.util.Arrays;
+
+@DesignerComponent(version = 1,
+    description = "A component that holds (x, y)-coordinate based data",
+    category = ComponentCategory.CHARTS,
+    iconName = "images/web.png")
+@SimpleObject
+public final class CoordinateData extends ChartDataBase {
```

Comment:
Since we now use the concept of tuples, perhaps it's sufficient to have only one Data Component type?

We could also introduce dropdowns for the Data component type if that is required.

Thoughts on this?